examples/tutorials/translations/german/Part 01 - The Basic Tools of Private Deep Learning.ipynb

```diff
@@ -0,0 +1,962 @@
+{
```

Comment:
<h1>Abschnitt 1.2 - Verwenden von Tensor-<em>pointern</em>&nbsp;</h1><p> </p><p>Das Senden und Empfangen von Tensoren von Bob ist zwar großartig, aber das ist kaum Deep Learning! Wir möchten in der Lage sein, Tensorberechnungen an entlegenen Tensoren durchzuführen. Glücklicherweise machen Tensor-<em>pointer</em> dies ziemlich einfach! Sie können einfach diese <em>pointer</em> wie normale Tensoren verwenden!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/3178/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>